src/Surfnet/Stepup/DateTime/DateTime.php

```diff
@@ -35,6 +35,7 @@ class DateTime implements Stringable
      * `DateTime::createFromString()`.
      */
     public const FORMAT = 'Y-m-d\\TH:i:sP';
+    public const MAIL_FORMAT = 'Y-m-d H:i:s';
```

Comment:
Should the timezone not be included? I do not see the original format that was used to check this change against.